### DISCUSSION TITLE
Why were the patch versions for CVE-2022-39383 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [kubevela/kubevela#6403] [Q&A]
We are a research team dedicated to Golang, have discovered that CVE-2022-39383 was addressed in commit 85489c63b4a3c2148229e5384af9e01e4e8471aa. However, upon analyzing the commit, we observed that the patch version (v1.7.0-alpha.1) was released after a lapse of over one month. We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users. We seek clarification on whether the delay might be attributed to:

1. Issues with testing and CI checking.
2. Other commits requiring inclusion in a single release.
3. By convention, infrequent release of versions.
4. Other reasons.
   We appreciate your attention to this matter and eagerly await your response. Thank you.

**Comment by wonderflow** [comment]
The patch was back-ported to v1.5.x and v1.6.x which was release with 1 week. https://github.com/kubevela/kubevela/pull/5000
